Yelp/Business.swift

```diff
@@ -84,11 +84,11 @@ class Business: NSObject {
         return businesses
     }
     
-    class func searchWithTerm(term: String, completion: ([Business]!, NSError!) -> Void) {
-        YelpClient.sharedInstance.searchWithTerm(term, completion: completion)
+    class func searchWithTerm(_ term: String, completion: @escaping ([Business]?, Error?) -> Void) {
```

Comment:
if we want to be very Swift 3.0....the signature would be something like `class func search(with term: String, completion...`